common/src/main/java/com/wynntils/models/containers/type/SearchableContainerType.java

```diff
@@ -5,33 +5,59 @@
 package com.wynntils.models.containers.type;
 
 import com.wynntils.core.text.StyledText;
+import com.wynntils.utils.wynn.ContainerUtils;
+import java.util.function.Predicate;
 import java.util.regex.Pattern;
+import java.util.stream.Stream;
+import net.minecraft.world.inventory.ChestMenu;
+import net.minecraft.world.item.ItemStack;
 
 public enum SearchableContainerType {
-    BANK(Pattern.compile("§0\\[Pg. (\\d+)\\] §8(.*)'s?§0 Bank"), Pattern.compile("§f§lPage \\d+§a >§2>§a>§2>§a>"), 8),
+    BANK(
+            java.util.regex.Pattern.compile("§0\\[Pg. (\\d+)\\] §8(.*)'s?§0 Bank"),
```

Comment:
```suggestion
            Pattern.compile("§0\\[Pg. (\\d+)\\] §8(.*)'s?§0 Bank"),
```
Same for the others :)

Reply:
Good catch!

Reply:
i don't even know how that got there xD